test/Microsoft.AspNetCore.Razor.Test/CodeGenerators/CSharpCodeGeneratorTest.cs

```diff
@@ -14,9 +17,12 @@ namespace Microsoft.AspNetCore.Razor.CodeGenerators
 {
     public class CSharpCodeGeneratorTest
     {
+        private static readonly string _testOutputDirectory = "TestFiles/CodeGenerator/Output/CSharpCodeGenerator";
```

Comment:
Discussed offline. This is actually used to find resources and `'\\'` isn't handled in that code. Fine as-is.